Tensile/TensileCreateLibrary.py

```diff
@@ -190,10 +190,18 @@ def isSupported(arch):
         print('hipcc:', ' '.join(compileArgs))
       subprocess.check_call(compileArgs)
 
+      infile = os.path.join(buildPath, objectFilename)
+      try:
+        bundlerArgs = [globalParameters["ClangOffloadBundlerPath"], "-type=o", "-inputs=%s" % infile, "-list"]
+        subprocess.check_output(bundlerArgs, stderr=subprocess.STDOUT).decode().split("\n")
+        # TODO - use the list of gfx entries returned as targets directly
+        hipVersion = "hipv4"
```

Comment:
since -list support is already in trunk and amd-stg-open branch but hipv4 change is not committed yet. So this change patch will break ePSDB. Even for a quick workaround, you still need to grep hipv4 in the output of 'clang-offloader-bundler -list'. If hipv4 is found, use hipv4, otherwise use hip.

Reply:
Done.